en/identity-server/7.0.0/docs/apis/restapis/scim2-users.yaml

```diff
@@ -523,6 +531,54 @@ components:
                 value:
                   type: string
                   example: Taylor
+    UserObjectPassInvite:
+      title: Invite the user to set their own password
+      type: object
+      properties:
+        schemas:
+          type: object
+          properties: {}
+          example: []
+        name:
+          type: object
+          properties:
+            givenName:
+              type: string
+              description: The `givenName` you specify will be entered for the First Name attribute in the user's profile.
+              example: Kim
+            familyName:
+              type: string
+              description: The `familyName` you specify will be entered for the Last Name attribute in the user's profile.
+              example: Berry
+        userName:
+          type: string
+          description: The `username` you specify will be used by the user for log in.
+          example: kim
+        emails:
+          type: array
+          description: Currently, the first email (without a specified type) is treated as the primary email and assigning a type to this email is not supported.
+          example:
+          - value: kim@gmail.com
+            primary: true
+          items:
+            type: object
+            properties: {}
+        urn:ietf:params:scim:schemas:extension:enterprise:2.0:User:
+          type: object
+          properties:
+            manager:
+              type: object
+              properties:
+                value:
+                  type: string
+                  example: Taylor
+        urn:scim:wso2:schema:
+          type: object
+          properties:
+            askPassword:
```

Comment:
This is not under `urn:scim:wso2:schema` schema in IS.
It should fall under `urn:ietf:params:scim:schemas:extension:enterprise:2.0:User` for IS
for Asgardeo, askPassword is under `urn:scim:wso2:schema`

Reply:
fixed with: https://github.com/wso2/docs-is/pull/4891/commits/c12a75e0702c62e0becb417c7b5e989dcad5cd49